packages/runner-ct/src/lib/ResizableBox.tsx

```diff
@@ -66,6 +66,7 @@ export const ResizableBox: React.FC<ResizableBoxProps> = ({
       {!disabled && (
         <div
           data-cy="resizer"
+          className="Resizer"
           style={{ cursor: 'col-resize', flexBasis: 11, width: 11 }}
```

Comment:
I think jira uses ew-resize as a cursor, which contains less clutter. I think we can trust that they did their due diligence on it.

```suggestion
          style={{ cursor: 'ew-resize', flexBasis: 11, width: 11 }}
```

Reply:
On another note, why is this in the JSX instead of in the SCSS file?

Reply:
jsx really should be updated. 
But I don't agree on the cursor. There is kind of standard for UX cursors and we can get info from there. I don't believe jira (actually any) developers.